parlai/core/torch_classifier_agent.py

```diff
@@ -98,16 +99,18 @@ def __init__(self, opt: Opt, shared=None):
         super().__init__(opt, shared)
 
         # set up classes
-        if opt.get('classes') is None and opt.get('classes_from_file') is None:
-            raise RuntimeError(
-                'Must specify --classes or --classes-from-file argument.'
-            )
         if not shared:
-            if opt['classes_from_file'] is not None:
-                with open(opt['classes_from_file']) as f:
-                    self.class_list = f.read().splitlines()
+            if opt.get('classes') is None:
+                raise RuntimeError('Must specify --classes argument.')
+            if os.path.isfile(opt['classes'][0]):
```

Comment:
was this not supported before with the `opt['classes_from_file']`?

Reply:
this was indeed supported with the 'classes_from_file' argument

Reply:
i suppose we could overload that argument, i'm not sure how i feel about it. if we decide to do that, then we should get rid of that argument